command/output.go

```diff
@@ -26,15 +27,19 @@ func (c *OutputCommand) Run(args []string) int {
 	}
 
 	args = cmdFlags.Args()
-	if len(args) != 1 || args[0] == "" {
+	var allOutputs bool = false
```

Comment:
Go style thing; declarations look like:

```go
// type implicitly set by value
foo := false
// type explicitly declared, gets zero value
var foo bool
```

^^ One or the other, rarely both.